vendor/vendor.json

```diff
@@ -513,6 +531,12 @@
 			"revision": "b001fa50d6b27f3f0bb175a87d0cb55426d0a0ae",
 			"revisionTime": "2018-11-28T10:09:59Z"
 		},
+		{
+			"checksumSHA1": "vW7IiPtoA4hQQ/ScHlbmRktY89U=",
+			"path": "github.com/tyler-smith/go-bip39",
+			"revision": "8e7a99b3e716f36d3b080a9a70f9eb45abe4edcc",
+			"revisionTime": "2016-06-29T16:38:56Z"
```

Comment:
This is quite old -- almost two years, and there have been quite a lot of commits to that repo since. Unless you have some good reason for it, you should probably use the latest revison instead, from october 2018

Reply:
done